### PR TITLE
Feature/add forms

### PR DESCRIPTION
**Pull request by ulisesedu7**
## Description
In this pull request, I've created the options to create new posts and to add comments and likes to each of the posts. 

**Task:** https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/forms_project_v1_1.md

This PR makes the following changes:
- Create new comments and likes controllers.
- Modify options to display the data.
- Add feature to create new posts

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix, feature, or code refactoring that would cause existing functionality not to work as expected)

## Checklist

- [X] My code follows the style guidelines of this project (no linters errors)
- [X] I have checked my code and corrected any misspellings
- [X] I have commented on my code
- [X] I have deleted any unused code as comments in my codebase

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -30,6 +30,18 @@
       </article>
     <% end %>
```
Comment (medaminedev66) [inline app/views/posts/index.html.erb:31]: [OPTIONAL] The posts index page does not show the posts of the user. I recommend fixing that for a better user experience.

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -16,22 +17,30 @@
...
-
         <% @posts.each do |post| %>
           <div class="post-card">
             <h3>Post #<%= post.id %>: <%= post.title %></h2>
             <p class="post-card-d"><%= post.text %></p>
             <p class="post-card-a">
-              Comments: <%= post.comment_counter %>, Likes: <%= post.like_counter %>
+              Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %>
             </p>
           </div>
```
Comment (medaminedev66) [inline app/views/users/show.html.erb:27]: [OPTIONAL] The user should be able to navigate to the post-show page by clicking on the post title. I recommend implementing that for a better user experience.

File: app/views/users/index.html.erb [hunk 3]
```diff
@@ -1,19 +1,20 @@
...
-          <div class="users-section-card">
+        <article class="users-section d-flex justify-content-center align-items-center">
+            <%= image_tag "user-regular.svg" %>
+
             <%= link_to user_path(user.id) do %>
-              <img src="<%= user.photo %>" alt="The representation of the user">
-              <h4><%= user.name %></h4>
-              <p>Number of Posts: <%= user.posts_counter %></p>
+              <div class="users-section-card d-flex">
+                <h4><%= user.name %></h4>
```
Comment (medaminedev66) [inline app/views/users/index.html.erb:14]: - [x] I tried to navigate to the page of the post I created but the posts/show page threw the following error:

image

Kindly fix the error, and ensure that the user can create comments and add likes.

File: app/views/users/show.html.erb [hunk 4]
```diff
@@ -1,8 +1,9 @@
...
-  <section class="users-section">
+  <section class="users-section d-flex justify-content-center flex-column align-items-center">
     <article class="users-info">
-      <div class="users-section-card">
-        <img src="<%= @user.photo %>" alt="Where the user face would be">
+      <div class="users-section-card d-flex">
+        <%= image_tag "user-regular.svg" %>
+
         <h4><%= @user.name %></h4>
         <p>Number of Posts: <%= @user.posts_counter %></p>
```
Comment (medaminedev66) [inline app/views/users/show.html.erb:8]: - [x] The posts_counter increases by 2 rather than 1 when I create a post. Please make that right.

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @ulisesedu7  😃, :wave:, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- There are no linter errors.
- You have a descriptive PR.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ulisesedu7** [comment]
@medaminedev66 Thanks for the feedback, I'll start working on it right now.

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ulisesedu7** [comment]
@CollinsTatang Thank you very much!!

**Pull request merged**